rasa/core/brokers/pika.py

```diff
@@ -221,38 +219,41 @@ def close_pika_connection(connection: "Connection") -> None:
 
 
 class PikaEventBroker(EventBroker):
+    """Pika-based event broker for publishing messages to RabbitMQ.
+
+    Attributes:
+        host: Pika host.
+        username: Username for authentication with Pika host.
+        password: Password for authentication with Pika host.
+        port: port of the Pika host.
+        queues: Pika queues to declare and publish to.
+        should_keep_unpublished_messages: Whether or not the event broker should
+            maintain a queue of unpublished messages to be published later in
+            case of errors.
+        raise_on_failure: Whether to raise an exception if publishing fails. If
+            `False`, keep retrying.
+        log_level: Logging level.
+
+    """
+
```

Comment:
```suggestion
```
If we are removing blank lines after google docstrings

Reply:
this is the class description, not a function description. this needs to be followed by a blank line (also black does this automatically). see google styleguide: http://google.github.io/styleguide/pyguide.html#384-classes

Reply:
Ah I see. Thanks for clarifying!